src/builders/rollup/utils.ts

```diff
@@ -15,6 +23,7 @@ export const DEFAULT_EXTENSIONS: string[] = [
 
 export function resolveAliases(ctx: BuildContext): Record<string, string> {
   const aliases: Record<string, string> = {
+    ...inferAliasesFromTsconfig(ctx),
```

Comment:
This should be opt-in

Reply:
Yes, do you suggest adding a new configuration for this feature?